TestDateFormat fails depending on timezone

## Bug Report

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

With master at 671fe7087607676cf296f2416ec9927c155520ed

```
make dev
```

With this timzone:
```
$ ls -l /etc/localtime 
lrwxrwxrwx. 1 root root 38 May 23 07:16 /etc/localtime -> ../usr/share/zoneinfo/Europe/Amsterdam
```

### 2. What did you expect to see? (Required)

Succesful test

### 3. What did you see instead (Required)

```
--- FAIL: TestDateFormat (0.00s)
    util_test.go:29: 
        	Error Trace:	/home/dvaneeden/dev/pingcap/tidb/br/pkg/stream/util_test.go:29
        	Error:      	Not equal: 
        	            	expected: "2022-07-15 13:14:39.534 +0200"
        	            	actual  : "2022-07-15 19:14:39.534 +0800"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-2022-07-15 13:14:39.534 +0200
        	            	+2022-07-15 19:14:39.534 +0800
        	Test:       	TestDateFormat
```

### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

master at 671fe7087607676cf296f2416ec9927c155520ed

--- a/br/pkg/stream/util_test.go
+++ b/br/pkg/stream/util_test.go
@@ -4,6 +4,7 @@ package stream
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 	"github.com/tikv/client-go/v2/oracle"
@@ -24,8 +25,9 @@ func TestDateFormat(t *testing.T) {
 		},
 	}
 
+	timeZone, _ := time.LoadLocation("Asia/Shanghai")
 	for _, ca := range cases {
-		date := FormatDate(oracle.GetTimeFromTS(ca.ts))
-		require.Equal(t, date, ca.target)
+		date := FormatDate(oracle.GetTimeFromTS(ca.ts).In(timeZone))
+		require.Equal(t, ca.target, date)
 	}
 }

